src/main/kotlin/org/rust/cargo/runconfig/CargoRunStateBase.kt

```diff
@@ -65,11 +65,7 @@ abstract class CargoRunStateBase(
      */
     fun startProcess(processColors: Boolean): ProcessHandler {
         val commandLine = cargo().toColoredCommandLine(environment.project, prepareCommandLine())
-        val handler = if (processColors) {
-            RsKillableColoredProcessHandler(commandLine)
-        } else {
-            KillableProcessHandler(commandLine)
-        }
+        val handler = if (processColors) RsProcessHandler(commandLine) else KillableProcessHandler(commandLine)
```

Comment:
It's ok that you still use `RsProcessHandler` here instead of `RsWslProcessHandler` on WSL?